snapcraft/file_utils.py

```diff
@@ -417,3 +418,15 @@ def get_resolved_relative_path(relative_path: str, base_directory: str) -> str:
     filename_relpath = os.path.relpath(filename_abspath, base_directory)
 
     return filename_relpath
+
+
+def _remove_readonly(func, path, excinfo):
+    """Try setting file to writeable if error occurs.
+    Known to be required on Windows."""
+    os.chmod(path, stat.S_IWRITE)
+    func(path)
+
+
+def rmtree(path: str) -> None:
+    """Cross-platform rmtree implementation."""
+    shutil.rmtree(path, onerror=_remove_readonly)
```

Comment:
please add an newline to get github to get rid of that ugly symbol :laughing:

Reply:
done!